dbms/src/Storages/StorageDisaggregated.h

```diff
@@ -154,5 +154,6 @@ class StorageDisaggregated : public IStorage
     const FilterConditions & filter_conditions;
 
     std::unique_ptr<DAGExpressionAnalyzer> analyzer;
+    std::unordered_set<uint64_t> store_id_blocklist;
```

Comment:
It will eventually be converted to unordered_set as an argument to a API in client-c. A better way(if necessay) is make store_id_blocklist into a sigleton.